lib/internal/test_runner/test.js

```diff
@@ -1089,7 +1090,10 @@ class Test extends AsyncResource {
           if (actual < threshold) {
             harness.success = false;
             process.exitCode = kGenericUserError;
-            reporter.diagnostic(nesting, loc, `Error: ${NumberPrototypeToFixed(actual, 2)}% ${name} coverage does not meet threshold of ${threshold}%.`);
+            reporter.diagnostic(nesting,
+                                loc,
+                                `${red}Error: ${NumberPrototypeToFixed(actual, 2)}% ${name} ` +
```

Comment:
Do we emit colors in the reporter events anywhere else? Doesn't doing so defeat the purpose of having a generic reporter interface?

I am in favor of highlighting this text in reporters that use colors though.

Reply:
Yes. In the coverage table, colors are used to show whether something has a close to 100% coverage or not

Reply:
Oh ☹️ We should not do that.

Reply:
> Yes. In the coverage table, colors are used to show whether something has a close to 100% coverage or not

If I'm not mistaken we colorise during the reporting, not directly in the emit.
I think that the problem here is that we shouldn't colorise the content of the emit

Reply:
maybe we should add a level paramater in diagnostics (i.e debug/info/warn/error) so reporters can implement coloring or other things

Reply:
In that case, this PR isn’t needed, and a new one should be open implementing that, right?